connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedKafkaCluster.java

```diff
@@ -748,9 +625,10 @@ public KafkaProducer<byte[], byte[]> createProducer(Map<String, Object> producer
         return producer;
     }
 
-    private static void putIfAbsent(final Map<String, Object> props, final String propertyKey, final Object propertyValue) {
-        if (!props.containsKey(propertyKey)) {
-            props.put(propertyKey, propertyValue);
-        }
+    private void addDefaultBrokerPropsIfAbsent(Properties brokerConfig, int numBrokers) {
+        brokerConfig.putIfAbsent(ServerConfigs.DELETE_TOPIC_ENABLE_CONFIG, "true");
```

Comment:
Do we need that? This is the default value

Reply:
Correct, we can remove. I think it was left over from a lateral refactor that moved from a custom `putIfAbsent` implementation to using the Java standard library.